src/pleasant-lcd.h

```diff
@@ -234,6 +236,50 @@ void lcd_fill_rect(uint16_t x,
                    uint16_t h,
                    lcd_color color);
 
+/* Touch ------------------------------------------------------------------- */
+typedef struct {
+  uint32_t x;
+  uint32_t y;
+} CAL_POINT;
```

Comment:
I'd prefer it if everything exported from this library started with `lcd_` and if structs were not `typedef`'d away. Like this:

```c
struct lcd_calibration_point {
  uint32_t x;
  uint32_t y;
};
```